scrapers/MetalBondage.yml

```diff
@@ -0,0 +1,39 @@
+name: Metal Bondage
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - metalbondage.com
+    scraper: sceneScraper
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //title/text()
+      Date:
+        selector: //div[@class="metabar"]/text()
+        postProcess:
+          - parseDate: January 2, 2006
+      Details:
+        selector: //div[@class="textcontent"]/p
+        concat: "\n\n"
+      Code:
+        selector: //div[@class="post-title fix"]//a/text()
+        postProcess:
+          - replace:
+              - regex: '(MB\d+)\s([\w-].*)'
+                with: $1
+      Performers:
+        Name:
+          selector: //div[@class="tags"]//a[@rel="tag"]/text()
+      Tags:
+        Name:
+          selector: //div[@class="tags"]//a[@rel="category tag"]/text()
+      Image: //div[@class="textcontent"]//img/@src
+      Studio:
+        Name:
+          fixed: "Metal Bondage"
+      URL: //link[@rel="canonical"]/@href
+
+driver:
+  useCDP: true
```

Comment:
As far as I can tell there's no need for a CDP here, the scraper works fine for me without it

Reply:
I thought I had it fail once without CDP but I can't reproduce now, so you might be right.